int64 class2repeat
---
bool success